ocaml/typing/typecore.ml

```diff
@@ -4414,41 +4415,6 @@ and type_expect_
       type_function ?in_function loc sexp.pexp_attributes env
                     expected_mode ty_expected_explained
                     l ~has_local ~has_poly:false [Exp.case pat body]
-
-
-  (* | Pexp_fun (l, None, {ppat_desc = Ppat_constraint (spat, {ptyp_desc = Ptyp_extension ({txt = "src_pos"; _}, _); _}); _}, sbody) ->
```

Comment:
Feel free to ignore this.. This comment was left at an early checkpoint in this branch